src/plugins/saved_objects_management/public/management_section/object_view/saved_object_view.tsx

```diff
@@ -152,11 +150,32 @@ export class SavedObjectEdition extends Component<
         buttonColor: 'danger',
       }
     );
-    if (confirmed) {
-      await savedObjectsClient.delete(type, id);
-      notifications.toasts.addSuccess(`Deleted '${object!.attributes.title}' ${type} object`);
-      this.redirectToListing();
+    if (!confirmed) {
+      return;
     }
+
+    const [{ success, error }] = await bulkDeleteObjects(http, [{ id, type }]);
+
+    if (!success) {
+      notifications.toasts.addDanger({
+        title: i18n.translate(
+          'savedObjectsManagement.objectView.unableDeleteSavedObjectNotificationMessage',
+          {
+            defaultMessage: `Failed to delete '{title}' {type} object`,
+            values: {
+              type,
+              title: object!.attributes.title,
```

Comment:
An object's title is computed by using the `getTitle` function associated with its type:

https://github.com/elastic/kibana/blob/4f4c160d565508a140b0e81f4237afdde53af983/packages/core/saved-objects/core-saved-objects-server/src/saved_objects_management.ts#L45-L49

Should use `object.meta.title` instead.

https://github.com/elastic/kibana/blob/3730dd0779ed8efd74aee88f57422781ec1ac122/src/plugins/saved_objects_management/common/types.ts#L18

E.g

https://github.com/elastic/kibana/blob/3730dd0779ed8efd74aee88f57422781ec1ac122/src/plugins/saved_objects_management/public/management_section/object_view/saved_object_view.tsx#L113